### PR TITLE
PLANET-3424: CPP - Logo color selector should update footer element colors too

### PR DESCRIPTION
**Pull request by pablocubico**
Ref: https://jira.greenpeace.org/browse/PLANET-3424

### CODE REVIEW

**Comment by comzeradd** [comment]
Also the force inside `phpcs` is complaining :D 

```
Use Yoda Condition checks, you must.
```

**Comment by pablocubico** [comment]
@comzeradd updated!

File: single-campaign.php [hunk 1]
```diff
@@ -35,9 +35,12 @@
 	$header_font = "font-family: {$pf} !important;";
 }
 
+$footer_links_color = 'light' == $post->campaign_logo_color ? '#FFFFFF' : '#1A1A1A';
```
Comment (angtheod) [inline single-campaign.php]: `yodatroll` also complains for not using strict comparison

**Comment by pablocubico** [comment]
@angtheod done!

**Pull request merged**